.github/actions/convertIssue/parse-issue-body.js

```diff
@@ -333,7 +333,7 @@ export async function parseIssueBody(githubIssueTemplateFile, body) {
       ),
       //unlikely to not be nrelop, could require manual changes
       token_prefix: "nrelop",
-      toekn_generate: combinedObject.autogen, //want to match the autogen above
+      token_generate: configObject['opcode']['autogen'], //want to match the autogen above
```

Comment:
@Abby-Wheelis Wait, I think I got this backwards. Shouldn't `token_generate` be NOT `autogen` ?

When `autogen` = true, tokens are generated on the phone. When `autogen` = false, tokens are pregenerated and this is when generating them on the admin dash makes sense

If that is correct, I will flip it and then merge this

```suggestion
      token_generate: !configObject['opcode']['autogen'], // only for pregenerated tokens
```